xml/operations-dns-designate_overview.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
```

Comment:
This one looks like it needs to be reformatted... (the other one looks clean though!)